Threat Intel section (#11270)
DataDog/documentation: content/en/security_platform/explorer.md

- `Related Issues` includes a list of other signals which contain the same group by values to assist with triaging the signal.

### Threat Intelligence

Datadog Security Monitoring offers threat intelligence feeds curated by threat intelligence partners. These feeds are constantly updated to include data about known suspicious activity (for example, IOCs), so you can quickly identify which potential threats to address.

> {{< img src="security_platform/security_monitoring/explorer/threat_intelligence.png" alt="Threat Intelligence in the Security Signals Explorer"  >}}

Reviewer: This image looks like it needs some blurring or cropping of sensitive info

Author: @jmassey-datadog Does this image contain sensitive info? It's being used in other places throughout the docs / blog.

> Datadog automatically operationalizes threat intelligence by analyzing all ingested logs that have relevant attributes. If there is an indicator of compromise in a log, such as an anonymized IP tied to a VPN, proxy, or Tor exit node, a `threat_intel` attribute is append to the log event to provide additional insights based on available intelligence.

Reviewer: ```suggestion
Datadog automatically operationalizes threat intelligence by analyzing all ingested logs that have relevant attributes. If a log contains a compromise indication, such as an anonymized IP tied to a VPN, proxy, or Tor exit node, a `threat_intel` attribute is append to the log event to provide additional insights based on available intelligence.
```

The query to see all threat intel matches in the Security Signals Explorer is `@threat_intel.indicators_matched:*`. The following are additional attributes to query for threat intelligence:

* `@threat_intel.results.category “anonymizer”, “scanner”`
* `@threat_intel.results.intention “malicious”, “unknown”`
* `@threat_intel.results.subcategory options "proxy", "tor", "vpn"`

>     **Note**: These are only provided by IPinfo.

Reviewer: ```suggestion
    **Note**: Proxy, Tor, and VPN subcategory attributes are provided only by threat intelligence partner IPinfo.
```
I think that's what you mean?
